lib/Haskell/Prim/String.agda

```diff
@@ -7,6 +7,10 @@ open import Haskell.Prim.Foldable
 
 --------------------------------------------------
 -- String
+-- This is _not_ the builtin String type of Agda;
+-- which is defined by postulates.
+-- fromString can be used to produce Haskell strings
+-- from that.
```

Comment:
Nice addition, but needs rephrasing:
1. No need for `;`, just say "This is _not_ the builtin String type of Agda that is defined by postulates".
2. `fromString` actually gives *Agda* strings, not Haskell ones, and in fact this type of strings in this module is what corresponds to Haskell strings. Thus, rephrase as "`fromString` can be used to convert back to builtin Agda strings"?